projects/ngrx.io/content/guide/component-store/install.md

```diff
@@ -0,0 +1,25 @@
+# Installation
```

Comment:
I've not read the other documentation of this PR (for now).
I just wanted to mention that we also have `ng-add` support 🙂

Reply:
Nice.
Done.